src/kernel/src/fs/dev/vnode.rs

```diff
@@ -88,10 +88,11 @@ impl crate::fs::VnodeBackend for VnodeBackend {
         let mode = dirent.mode();
         let size = match vn.ty() {
             VnodeType::Directory(_) => 512,
-            VnodeType::Character => 0,
+            VnodeType::Link => todo!(), /* TODO: strlen(dirent.de_symlink) */
+            _ => 0,
         };
 
-        Ok(VnodeAttrs::new(*uid, *gid, *mode, size))
+        Ok(VnodeAttrs::new(*uid, *gid, *mode, size, 0))
```

Comment:
Does the PS4 set fsid to zero here or we just use zero to represents not yet implemented? If it is the latter case better to replace the whole line with `todo` macro instead.